src/components/Tool/MergePeople/MergePeople.tsx

```diff
@@ -93,40 +102,112 @@ const MergePeople: React.FC<Props> = ({
     }
   };
 
-  const mergePeople = async () => {
-    const mergeActions = Object.entries(actions).filter(
-      (action) => action[1].action === 'merge',
-    );
-    if (mergeActions.length) {
-      const winnersAndLosers: { winnerId: string; loserId: string }[] =
-        mergeActions.map((action) => {
-          return { winnerId: action[0], loserId: action[1].mergeId || '' };
-        });
-      await peopleMerge({
-        variables: {
-          input: {
-            winnersAndLosers,
-          },
-        },
-        update: (cache) => {
-          // Delete the loser people and remove dangling references to them
-          winnersAndLosers.forEach((person) => {
-            cache.evict({ id: `Person:${person.loserId}` });
+  const handleBulkUpdateDuplicates = async () => {
```

Comment:
The same goes for these other functions, you might be able to create it once and use it on both pages.